pydata_sphinx_theme/locale/es/LC_MESSAGES/sphinx.po

```diff
@@ -0,0 +1,83 @@
+# Spanish translations for pydata-sphinx-theme.
+# Copyright (C) 2021 ORGANIZATION
+# This file is distributed under the same license as the pydata-sphinx-theme
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: pydata-sphinx-theme 0.6.1\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2021-04-22 16:46-0400\n"
+"PO-Revision-Date: 2021-04-22 16:46-0400\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: es\n"
+"Language-Team: es <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.9.0\n"
+
+#: pydata_sphinx_theme/docs-navbar.html:9
+msgid "Toggle navigation"
+msgstr "Alternar navegación"
+
+#: pydata_sphinx_theme/icon-links.html:14
+msgid "GitHub"
+msgstr "GitHub"
+
+#: pydata_sphinx_theme/icon-links.html:15
+msgid "GitLab"
+msgstr "GitLab"
+
+#: pydata_sphinx_theme/icon-links.html:16
+msgid "Bitbucket"
+msgstr "Bitbucket"
+
+#: pydata_sphinx_theme/icon-links.html:17
+msgid "Twitter"
+msgstr "Twitter"
+
+#: pydata_sphinx_theme/layout.html:7
+msgid "previous page"
+msgstr "página anterior"
+
+#: pydata_sphinx_theme/layout.html:10
```

Comment:
Does this mean that any time somebody moves an HTML template, or adds to one so that line numbers change, these files must be updated manually? Can we document that process?

Reply:
These comments are just to help translators locate the original string. There are so few strings that this assistance isn't really necessary. If we want to keep them up-to-date, then yes, they will have to be updated in such cases.

Reply:
Ah ok - I didn't realize that those didn't have a functional use.

Reply:
I've added a short paragraph to explain the comments in 05d6effe